app/products/calls/store/reducers/calls.ts

```diff
@@ -53,6 +53,11 @@ function calls(state: Dictionary<Call> = {}, action: GenericAction) {
         nextState[newCall.channelId] = newCall;
         return nextState;
     }
+    case CallsTypes.RECEIVED_CALL_ENDED: {
+        const nextState = {...state};
+        delete nextState[action.data.channelId];
+        return nextState;
+    }
     case CallsTypes.RECEIVED_CALL_FINISHED: {
```

Comment:
@cpoile I couldn't see this used anywhere other than tests. Is it something we can get rid of?

Reply:
Yes, we can get rid of it